crates/cxx-qt-build/Cargo.toml

```diff
@@ -22,6 +22,9 @@ quote.workspace = true
 qt-build-utils.workspace = true
 codespan-reporting = "0.11"
 version_check = "0.9"
+scratch = "1.0"
+serde = {version = "1.0", features = ["default", "derive"]}
```

Comment:
elsewhere we have a space around the braces, so `serde = { version = "1.0", features = ["default", "derive"] }`

Reply:
Done